src/Region/RegionHandler.cc

```diff
@@ -2559,19 +2540,15 @@ RegionState RegionHandler::GetTemporaryRegionState(std::shared_ptr<casacore::Coo
     if (corner.empty()) {
         return RegionState();
     }
-    point.set_x(corner[0]);
-    point.set_y(corner[1]);
-    control_points[0] = point;
+    control_points[0] = Message::Point(corner[0], corner[1]);
```

Comment:
For consistency I think we should add one more utility function for `std::vector<double>`.